p2pserver/actor/server/actor.go

```diff
@@ -51,15 +51,15 @@ func (this *P2PActor) Start() (*actor.PID, error) {
 func (this *P2PActor) Receive(ctx actor.Context) {
 	switch msg := ctx.Message().(type) {
 	case *actor.Restarting:
-		log.Info("p2p actor restarting")
+		log.Debug("[p2p]actor restarting")
```

Comment:
`restarting` means panic encountered, so debug level is not enough

Reply:
change log to Warn except the start state